sys/net/gnrc/transport_layer/tcp/gnrc_tcp_option.c

```diff
@@ -51,7 +51,9 @@ int _option_parse(gnrc_tcp_tcb_t *tcb, tcp_hdr_t *hdr)
                 continue;
 
             case TCP_OPTION_KIND_MSS:
-                if (opt_left < TCP_OPTION_LENGTH_MIN || option->length > opt_left || option->length != TCP_OPTION_LENGTH_MSS) {
+                if (opt_left < TCP_OPTION_LENGTH_MIN || option->length > opt_left ||
+                    option->length != TCP_OPTION_LENGTH_MSS) {
+                    
```

Comment:
Unrelated change, but okay.

Reply:
Was > 100 chars

Reply:
Usually we put unrelated beatification piggy-backs like this in a separate commit ;-).